modules/mono/glue/GodotSharp/GodotSharp/Core/NativeInterop/NativeFuncs.cs

```diff
@@ -510,7 +510,7 @@ public static partial void
 
         public static partial void godotsharp_print_rich(in godot_string p_what);
 
-        internal static partial void godotsharp_printerr(in godot_string p_what);
+        public static partial void godotsharp_printerr(in godot_string p_what);
```

Comment:
`NativeFuncs` methods should have never been public, I'd rather not expose more of its methods.

Reply:
I do agree, I'm actually thinking about keeping a separate reference of the native `print_error` and `print_line` function pointers directly inside the `Main.cs` and using that for messaging, but I don't think that's a healthy approach...

Reply:
I think I should just revert the changes in `NativeFuncs`, and retain only the `Main.WriteLine` implementation.